### PR TITLE
[RDR] Add Initial Bownfield changes for RDR

### PR DESCRIPTION
**Pull request by prsurve**

### CODE REVIEW

File: ocs_ci/framework/conf/default_config.yaml [hunk 1]
```diff
@@ -264,6 +264,9 @@ ENV_DATA:
   multus_cluster_net_type: "macvlan"
   multus_cluster_net_mode: "bridge"
 
+  #RDR Green filed
```
Comment (shylesh) [inline ocs_ci/framework/conf/default_config.yaml:267]:
```suggestion
  #RDR Green field
```

**Review by shylesh — COMMENTED** [review]
A minor comment otherwise LGTM

File: ocs_ci/framework/conf/default_config.yaml (hunk 1 above)
Comment (prsurve) [reply ocs_ci/framework/conf/default_config.yaml]: I have updated in new comment

**Review by shylesh — DISMISSED** [review]
LGTM

**Review by am-agrawa — APPROVED** [review]

**Review by keesturam — APPROVED** [review]

**Review by am-agrawa — APPROVED** [review]

**Pull request merged**